queries.sql

```diff
@@ -7,4 +7,41 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon','Pikachu');
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = TRUE;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ]  Let's be specific on what we're selecting from the animals table. You updated the `species` column in the animals table so it would be nice to select only `species` from the animals table to verify that change was made and also that change was undone instead of selecting everything in the animals table. hint: `SELECT species from animals;`

- [ ]  Kindly verify that the change was undone after rollback.